hoomd/dem/pair.py

```diff
@@ -295,6 +295,9 @@ class SWCA(hoomd.md.force._force, _DEMBase):
     :math:`r_{cut}=2^{1/6}\sigma` and :math:`\sigma=2\cdot r` were
     applied between the contact points of each pair of particles.
 
+    TODO::
```

Comment:
Note how above this it states:
"The effect is as if a
    :py:class:`hoomd.md.pair.SLJ` interaction with
    :math:`r_{cut}=2^{1/6}\sigma` and :math:`\sigma=2\cdot r` were
    applied between the contact points of each pair of particles."

Reply:
DEM is being replaced by ALJ in v3.0.

Reply:
Should I delete this comparison?